### PR TITLE
feat(#133): Add `MeasuredXsl` And Tweak `line-is-absent` Check

### PR DESCRIPTION
**Pull request by volodya-lombrozo**
In this PR I've:

1) Added `MeasuredXsl` that can measure time of all `xsl` transformations and prints messages into a log:
```
18:35:05 [WARN] org.eolang.lints.MeasuredXsl: XSL transformation 'sparse-decoration' took 473ms, whereas threshold is 100ms
```

2) I tweaked `line-is-absent` check:

From `ProgramTest#lintsLargeJnaClass(@Mktmp final Path home)`:

Before:
```
org.eolang.lints.MeasuredXSL: XSL transformation 'line-is-absent' took 4s, whereas threshold is 100ms
```
After:
```
17:51:22 [WARN] org.eolang.lints.MeasuredXSL: XSL transformation 'line-is-absent' took 117ms, whereas threshold is 100ms
```

___

Benchmark

Before:

```
Input: com/sun/jna/Pointer.class
Size of .class: 22Kb (22Kb bytes)
Size of .xmir after disassemble: 1Mb (1Mb bytes, 29630 lines)
Lint time: 9s (9332 ms)
```

After:

```
Input: com/sun/jna/Pointer.class
Size of .class: 22Kb (22Kb bytes)
Size of .xmir after disassemble: 1Mb (1Mb bytes, 29630 lines)
Lint time: 7s (7200 ms)
```

Realted to #133.

### CODE REVIEW

**Comment by volodya-lombrozo** [comment]
@yegor256 Could you have a look, please?

File: src/main/java/org/eolang/lints/MeasuredXsl.java [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+import com.jcabi.log.Logger;
+import com.jcabi.xml.Sources;
+import com.jcabi.xml.XML;
+import com.jcabi.xml.XSL;
+
+/**
+ * XSL that measures the time of transformation.
+ * @since 0.1
+ */
+public final class MeasuredXsl implements XSL {
```
Comment (yegor256) [inline src/main/java/org/eolang/lints/MeasuredXsl.java:35]: @volodya-lombrozo maybe it's better to put it into `test/` directory? It's only for tests, right?

Comment (volodya-lombrozo) [reply src/main/java/org/eolang/lints/MeasuredXsl.java:35]: @yegor256 I can't place it into `test/` directory, since I use in in `LintByXsl` class.

Comment (yegor256) [reply src/main/java/org/eolang/lints/MeasuredXsl.java:35]: @volodya-lombrozo let's make it package private then

**Review by yegor256 — CHANGES_REQUESTED** [review]
@volodya-lombrozo let's make this class package-private, since it's not part of external API of this library

**Comment by yegor256** [comment]
@rultor merge

**Comment by yegor256** [comment]
@volodya-lombrozo thanks!

**Pull request merged**